app/views/tools/index.html.erb

```diff
@@ -1,12 +1,75 @@
-<p>This is totally working</p>
 <div class= "container m-0 mw-100">
   <div class= "row m-0">
     <div class="col-xs-12 col-sm-6 m-0 w-100">
       <h2>tool researched</h2>
-      <p>col6</p>
+      <h5>paragraph 2</h5>
+      <div class= "form-row align-items-center my-4">
+        <div class= "form-group col-4 m-1">
+          <p>Category</p>
+          <select class="form-control ">
+            <option>Gardening</option>
```

Comment:
We are wondering if we can write this in ruby. Either with an each iteration over categories or with somethin g like this <%= f.input :category, collection: Restaurant::CATEGORIES %>

Reply:
Yes! I was just making a model that we can replace with Ruby when we have seeded the db and have some tools to show.